sites/cnfpt/requirements/base.txt

```diff
@@ -6,7 +6,7 @@ django-storages==1.10.1
 dockerflow==2020.10.0
 gunicorn==20.0.4
 psycopg2-binary==2.8.6
-richie==2.0.0-beta.22
+richie==2.0.1
 sentry-sdk==0.19.1
```

Comment:
Relevant to synchronize package versions with richie ?

- **factory-boy:** `3.2.0`
- **sentry-dsk**: `0.19.5`

Reply:
yes good idea